lib/util/foretoken.ex

```diff
@@ -0,0 +1,22 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule Antikythera.Foretoken do
+  @moduledoc """
+  A thin wrapper around [`Foretoken`](https://github.com/skirino/foretoken) to avoid collisions of bucket name between executor pool.
+  For details, refer to [Foretoken's documentation](https://hexdocs.pm/foretoken/api-reference.html).
+  """
+
+  @doc """
+  Usage of this function is same as `Foretoken.token/4` except that an executor pool argument is added.
```

Comment:
- is same => is the same
- `Foretoken.token/4` => `take/4`
- "an executor pool argument is added" => "an executor pool ID is required as an argument"

Reply:
Fixed in 1019fb721e18ae6f481e67446d31292a70595834